apis/v1alpha1/iam_request_wrapper.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import "time"
+
+// IAMRequestWrapper wraps the IAMRequest and adds additional fields such as
+// duration.
+type IAMRequestWrapper struct {
+	// Request contains IAM binding information.
+	Request *IAMRequest
```

Comment:
Just to confirm IIUC, you meant the following? Will it confuse people that duration must also be provided in the PR (instead of by the PR label), or you meant to put IAMRequestWrapper and IAMRequest in the same file.
```go
type IAMRequest struct {
	ResourcePolicies []*ResourcePolicy `yaml:"policies,omitempty"`
        Duration time.Duration
}
```

Reply:
discussed offline